runtime/src/main/java/org/corfudb/runtime/exceptions/WrongClusterException.java

```diff
@@ -0,0 +1,30 @@
+package org.corfudb.runtime.exceptions;
+
+import java.util.UUID;
+import javax.annotation.Nullable;
+import lombok.Getter;
+import org.corfudb.util.UuidUtils;
+
+public class WrongClusterException extends RuntimeException {
+
+    /** The cluster we expected to connect to. */
+    @Getter
+    UUID expectedCluster;
+
+    /** The cluster we actually ended up connecting to. */
+    @Getter
+    UUID actualCluster;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "actualCluster" field final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1165)